src/config/scrivitoSites.ts

```diff
@@ -3,9 +3,13 @@ import { scrivitoTenantId, isMultitenancyEnabled } from './scrivitoTenants'
 
 const location = typeof window !== 'undefined' ? window.location : undefined
 
+const BLACKLISTED_SITE_ID = 'mailing-app'
+
 export function baseUrlForSite(siteId: string): string | undefined {
```

Comment:
Instead of completely blacklisting `mailing-app` we should return `https://mailing.neoletter.com/<instanceId>` when `baseUrlForSite('mailing-app')` is requested.

`siteForUrl` with an url, that starts with `https://mailing.neoletter.com/<instanceId>` should return `mailing-app` if I'm not mistaken.